packages/nimble-components/src/table-column/base/tests/table-column.spec.ts

```diff
@@ -48,6 +49,22 @@ describe('TableColumn', () => {
         expect(element.columnInternals.currentPixelWidth).toBe(200);
     });
 
+    it('setting sortDirection sets columnInternals.currentSortDirection', async () => {
```

Comment:
I think there should be some more tests in this file around `sorting-disabled`.

Reply:
Added the test mentioned in your other comment, plus one that ensures that currentSortIndex/currentSortDirection are undefined when sortingDisabled is true. Those, coupled with the tests that set sortingDisabled in table-sorting.spec.ts, cover the sortingDisabled cases I can think of, but if you think of other test cases to add let me know.